.github/actions/runleaks/README.md

```diff
@@ -0,0 +1,163 @@
+# runleaks
+
+[![Scan Action Logs](https://github.com/JosiahSiegel/runleaks/actions/workflows/main.yml/badge.svg?branch=main)](https://github.com/JosiahSiegel/runleaks/actions/workflows/main.yml)
+
+Leverages [git-secrets](https://github.com/awslabs/git-secrets) to identify potential leaks in GitHub action run logs.
+
+ * Common Azure and Google Cloud patterns are available, thanks to fork [msalemcode/git-secrets](https://github.com/msalemcode/git-secrets).
+
+
+## Inputs
+```yml
+  github-token:
+    description: 'Token used to login to GitHub'
+    required: true
+  repo:
+    description: 'Repo to scan run logs for exceptions'
+    required: false
+    default: ${{ github.repository }}
+  run-limit:
+    description: 'Limit on how many runs to scan'
+    required: false
+    default: '50'
+  min-days-old:
+    description: 'Min age of runs in days'
+    required: false
+    default: '0'
+  max-days-old:
+    description: 'Max age of runs in days'
+    required: false
+    default: '3'
+  patterns-path:
+    description: 'Patterns file path'
+    required: false
+    default: ".runleaks/patterns.txt"
+  exclusions-path:
+    description: 'Excluded patterns file path'
+    required: false
+    default: ".runleaks/exclusions.txt"
+  fail-on-leak:
+    description: 'Fail action if leak is found'
+    required: false
+    default: true
+```
+
+## Outputs
+```yml
+  exceptions:
+    description: 'Json output of run logs with exceptions'
+  count:
+    description: 'Count of exceptions'
+```
+
+## Usage
+ * Note: [GitHub rate limits](#rate-limits)
+```yml
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Scan run logs
+        uses: josiahsiegel/runleaks@v1
```

Comment:
needs upddate

Reply:
All these issues are not documented and we are in the process of reaching out to Mr. Josiah Siegel and ask for his assistance in figuring out the different environment and secrets we need to setup in this project.